docs/tempo/website/setup/tanka.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Deploy on Kubernetes with Tanka
+weight: 200
+---
+
+# Deploy on Kubernetes with Tanka
+
+Using this deployment guide, you can deploy Tempo to Kubernetes using a Jsonnet library and [Grafana Tanka](https://tanka.dev) to create a development cluster or sand-boxed environment.
+This procedure uses MinIO to provide object storage regardless of the Cloud platform or on-premise storage you use.
+In a production environment, you can use your cloud provider’s object storage service to avoid the operational overhead of running object storage in production.
+
+This demo configuration includes [metrics-generator](https://grafana.com/docs/tempo/next/configuration/#metrics-generator).
+
+To set up Tempo using Kubernetes with Tanka, you need to:
+
+1. Configure Kubernetes and install Tanka
+1. Set up the Tanka environment
+1. Install libraries
+1. Deploy MinIO object storage and configure metrics-generator
+1. Deploy Tempo with the Tanka command
+
+>**Note**: This configuration is not suitable for a production environment but can provide a useful way to learn about Tempo.
+
+## Before you begin
+
+To deploy Tempo to Kubernetes with Tanka, you need:
+
+  * A Kubernetes cluster with at least 40 CPUs and 46GB of memory for the default configuration. Small ingest or query volumes could use a far smaller configuration.
+  * `kubectl`
```

Comment:
Version requirements?

Reply:
That depends on the API version of Kubernetes I suppose.

Reply:
@zalegrala What do you think about this? 
```suggestion
  * `kubectl`  (version depends upon the API version of Kubernetes)
```

Reply:
Maybe add `in your cluster` to the end. But yes, all dependent on the API version.